workflow/tests/base_results/results.csv

```diff
@@ -61,7 +61,7 @@ base-dhw-combi-tankless-outside.xml,50.86,50.86,21.419,21.419,29.441,0.0,0.0,0.0
 base-dhw-desuperheater-gshp.xml,36.28,36.28,36.28,36.28,0.0,0.0,0.0,0.0,0.0,0.0,4.822,0.0,0.397,1.958,0.797,7.03,0.0,0.0,4.51,0.0,0.334,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.319,0.365,1.513,1.528,0.0,2.114,8.374,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,17.974,10.202,9.233,0.615,2.426,0.0,0.0,0.0,3188.4,2129.2,20.613,15.055,0.0,3.616,3.648,0.514,7.129,0.633,-2.027,0.0,0.0,6.66,-0.118,4.809,0.0,0.729,0.0,3.145,-11.085,0.0,0.05,-0.419,-0.045,2.939,-0.015,9.969,0.0,0.0,-5.619,-0.057,-1.116,-7.102,-0.157,0.0,1.632,10.364,1354.8,997.6,11407.7,2617.7,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
-base-dhw-desuperheater-hpwh.xml,53.481,53.481,28.795,28.795,24.686,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.407,3.629,0.658,2.824,0.0,0.0,4.51,0.0,0.334,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.319,0.365,1.513,1.528,0.0,2.114,8.374,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,24.686,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,23.117,11.051,9.25,1.807,2.502,0.0,0.0,0.0,1878.1,3056.1,22.763,18.476,0.0,3.542,3.643,0.513,7.11,0.629,-2.118,0.0,0.0,6.682,-0.1,4.798,0.0,0.727,0.0,5.241,-7.901,0.0,0.045,-0.376,-0.039,3.064,-0.006,10.029,0.0,0.0,-5.4,-0.042,-1.072,-6.622,-0.148,0.0,2.744,9.227,1354.7,997.5,11327.0,2599.2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+base-dhw-desuperheater-hpwh.xml,53.481,53.481,28.795,28.795,24.686,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.407,3.629,0.658,2.824,0.0,0.0,4.51,0.0,0.334,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.319,0.365,1.513,1.528,0.0,2.114,8.374,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,24.686,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,23.117,11.051,9.25,1.807,2.503,0.0,0.0,0.0,1878.1,3056.1,22.763,18.475,0.0,3.542,3.643,0.513,7.11,0.629,-2.118,0.0,0.0,6.682,-0.1,4.798,0.0,0.727,0.0,5.241,-7.901,0.0,0.045,-0.376,-0.039,3.064,-0.006,10.029,0.0,0.0,-5.4,-0.042,-1.072,-6.622,-0.148,0.0,2.744,9.227,1354.7,997.5,11327.9,2599.4,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
```

Comment:
Why did some existing files change slightly?

Reply:
I removed a few uses of `round()`. I think maybe because of that.